extensions/vscode/src/test/unit-test-utils/factories.ts

```diff
@@ -44,7 +44,7 @@ export const preContentRecordFactory = Factory.define<PreContentRecord>(
     saveName: `Report ${sequence}`,
     createdAt: new Date().toISOString(),
     localId: "",
```

Comment:
Similar note about the removal of `localId`

Reply:
thanks again.